lib/tests/group.rs

```diff
@@ -311,7 +311,7 @@ async fn select_multi_aggregate() -> Result<(), Error> {
 			},
 			{
 				group: 2,
-				one: 7.6,
+				one: 7.6000000000000005,
```

Comment:
floating-point precision's nightmare.
I am wondering how we could avoid that.
Shall we convert back the value to an f32 when we print out the value?

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=7b21ea61932a4f3c8d46fd82c124156b

Reply:
>  Shall we convert back the value to an f32 when we print out the value?

You mean as opposed to f64 or Decimal? Right now the entire calculation is in f64 (no longer any Decimal involved) so f32 would lose precision in many cases. It could cause some unexpected results similar to Sqlite3:
```
sqlite> select (0.3) = 0.3;
1
sqlite> select 0.1 + 0.2;
0.3
sqlite> select (0.1 + 0.2) = 0.3;
0 <----- ?!
```


<details>
<summary>More details</summary>
In this particular case, we have: 3.2f64 + 4.4f64 = 7.6000000000000005f64 (which is true whether you use `math::sum` or `+`)
But, for f64 to display as 7.6, you need 7.599999999999999644f64 (which is the next lower value)
</details>